composer.json

```diff
@@ -83,6 +85,7 @@
         "drupal/search_api": "^1.23",
         "drupal/search_api_autocomplete": "^1.6",
         "drupal/smart_date": "^3.5",
+        "drupal/subgroup": "^1.0",
```

Comment:
Is this needed? Doesn't seem to be enabled.

Reply:
Removed subgroup module from composer